src/duckstation-qt/controllerledsettingsdialog.ui

```diff
@@ -1,83 +1,112 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <ui version="4.0">
- <class>ControllerLEDSettingsDialog</class>
- <widget class="QDialog" name="ControllerLEDSettingsDialog">
-  <property name="geometry">
-   <rect>
-    <x>0</x>
-    <y>0</y>
-    <width>501</width>
-    <height>108</height>
-   </rect>
-  </property>
-  <property name="windowTitle">
-   <string>Controller LED Settings</string>
-  </property>
-  <layout class="QGridLayout" name="gridLayout">
-   <item row="0" column="0">
-    <widget class="QGroupBox" name="groupBox">
-     <property name="title">
-      <string>SDL-0 LED</string>
-     </property>
-     <layout class="QVBoxLayout" name="verticalLayout_4">
-      <item>
-       <widget class="ColorPickerButton" name="SDL0LED"/>
-      </item>
-     </layout>
-    </widget>
-   </item>
-   <item row="0" column="1">
-    <widget class="QGroupBox" name="groupBox_2">
-     <property name="title">
-      <string>SDL-1 LED</string>
-     </property>
-     <layout class="QVBoxLayout" name="verticalLayout_3">
-      <item>
-       <widget class="ColorPickerButton" name="SDL1LED"/>
-      </item>
-     </layout>
-    </widget>
-   </item>
-   <item row="0" column="2">
-    <widget class="QGroupBox" name="groupBox_3">
-     <property name="title">
-      <string>SDL-2 LED</string>
-     </property>
-     <layout class="QVBoxLayout" name="verticalLayout_2">
-      <item>
-       <widget class="ColorPickerButton" name="SDL2LED"/>
-      </item>
-     </layout>
-    </widget>
-   </item>
-   <item row="0" column="3">
-    <widget class="QGroupBox" name="groupBox_4">
-     <property name="title">
-      <string>SDL-3 LED</string>
-     </property>
-     <layout class="QVBoxLayout" name="verticalLayout">
-      <item>
-       <widget class="ColorPickerButton" name="SDL3LED"/>
-      </item>
-     </layout>
-    </widget>
-   </item>
-   <item row="1" column="0" colspan="4">
-    <widget class="QDialogButtonBox" name="buttonBox">
-     <property name="standardButtons">
-      <set>QDialogButtonBox::Close</set>
-     </property>
-    </widget>
-   </item>
-  </layout>
- </widget>
- <customwidgets>
-  <customwidget>
-   <class>ColorPickerButton</class>
-   <extends>QPushButton</extends>
-   <header>colorpickerbutton.h</header>
-  </customwidget>
- </customwidgets>
- <resources/>
- <connections/>
+	<class>ControllerLEDSettingsDialog</class>
+	<widget class="QDialog" name="ControllerLEDSettingsDialog">
+		<property name="geometry">
+			<rect>
+				<x>0</x>
+				<y>0</y>
+				<width>501</width>
+				<height>128</height>
+			</rect>
+		</property>
+		<property name="windowTitle">
+			<string>Controller LED Settings</string>
+		</property>
```

Comment:
I'm not really clear what's going on with the spacing, as it doesn't look like this when pulled locally. If this is a blocker let me know.

Reply:
Did you edit it with Qt designer? It looks like it's been changed from 1 space indent to tabs.

Reply:
Ah, yeah that was the issue. Visual Studio was reformatting it after viewing/saving for me.